### PR TITLE
Initial NonProd Charts For MI ADF SHIR Setup

### PR DESCRIPTION
**Pull request by tchow8**
A release chart with default values for sbox.
A sbox environment chart as example usage for a Helm Release.

### CODE REVIEW

File: k8s/environments/sbox/cluster-00/mi/mi-adf-shir.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  namespace: mi
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.job: glob:dev-*
+spec:
+  releaseName: mi-adf-shir
+  chart:
+    git: git@github.com:hmcts/shared-services-flux
+    path: k8s/release/mi/mi-adf-shir
+    ref: setup-sbox-shir-charts
```
Comment (tchow8) [inline k8s/environments/sbox/cluster-00/mi/mi-adf-shir.yaml:15]: Change ref to master after tests complete.

File: k8s/release/mi/mi-adf-shir/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        - name: {{ .Values.keyVaultName }}-secrets-store
+          csi:
+            driver: secrets-store.csi.k8s.io
+            readOnly: true
+            volumeAttributes:
+              secretProviderClass: {{ .Values.appName }}-secret-provider
+      containers:
+        - name: mi-adf-shir
+          image: {{ .Values.image }}
+          env:
```
Comment (qzhou-hmcts) [inline k8s/release/mi/mi-adf-shir/templates/deployment.yaml:29]: these needs to be CSI secrets

Comment (qzhou-hmcts) [reply k8s/release/mi/mi-adf-shir/templates/deployment.yaml:29]: Ignore this

**Review by qzhou-hmcts — APPROVED** [review]

File: k8s/environments/dev/cluster-00/mi/mi-adf-shir.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    path: k8s/release/mi/mi-adf-shir
+    ref: master
+  values:
+    labels:
+      app.kubernetes.io/instance: mi-adf-shir-deployment
+      app.kubernetes.io/name: mi-adf-shir-deployment
+    image: sdshmctspublic.azurecr.io/mi/adf-integration-runtime:dev-814f45b
+    secretsMountPath: ''
+    environment: "dev"
+    resourcegroup: "mi-dev-rg"
```
Comment (qzhou-hmcts) [inline k8s/environments/dev/cluster-00/mi/mi-adf-shir.yaml:23]: resourceGroup

File: k8s/environments/dev/common/mi/mi-azure-functions.yaml [hunk 4]
```diff
@@ -18,6 +18,10 @@ spec:
       app.kubernetes.io/instance: mi-azure-functions-deployment
       app.kubernetes.io/name: mi-azure-functions-deployment
     image: sdshmctspublic.azurecr.io/mi/azure-functions:pr-4-60887fb
+    env:
+      ADF_RESOURCEGROUP: mi-dev-rg
+      ADF_NAME: mi-ingestion-adf-dev
+      ADF_STORAGEACCOUNTNAME: miauditdev
     environment: "dev"
     resourcegroup: "mi-dev-rg"
```
Comment (qzhou-hmcts) [inline k8s/environments/dev/common/mi/mi-azure-functions.yaml:26]: resourceGroup

File: k8s/environments/test/common/mi/mi-azure-functions.yaml [hunk 5]
```diff
@@ -23,6 +23,8 @@ spec:
       ADF_RESOURCEGROUP: mi-test-rg
       ADF_NAME: mi-ingestion-adf-test
       ADF_STORAGEACCOUNTNAME: miaudittest
+      ADF_TENANTID: "{{ .Values.tenantId }}"
```
Comment (qzhou-hmcts) [inline k8s/environments/test/common/mi/mi-azure-functions.yaml:26]: is this needed?

**Pull request merged**